crates/primitives/src/block.rs

```diff
@@ -203,6 +203,11 @@ impl SealedBlock {
             self.ommers.iter().map(Header::size).sum::<usize>() + self.ommers.capacity() * std::mem::size_of::<Header>() +
             self.withdrawals.as_ref().map(|w| w.iter().map(Withdrawal::size).sum::<usize>() + w.capacity() * std::mem::size_of::<Withdrawal>()).unwrap_or(std::mem::size_of::<Option<Vec<Withdrawal>>>())
     }
+
+    /// Calculates the total gas used by blob transactions in the sealed block.
+    pub fn calculate_blob_gas_used(&self) -> u64 {
```

Comment:
nit: same name as for Transaction
```suggestion
    pub fn blob_gas_used(&self) -> u64 {
```